frontend/.eslintrc.json

```diff
@@ -0,0 +1,56 @@
+{
+    "env": {
+      "browser": true,
+      "amd": true,
+      "es2021": true
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended",
+      "plugin:react-hooks/recommended",
+      "plugin:import/errors",
+      "plugin:import/warnings",
+      "plugin:prettier/recommended",
+      "plugin:cypress/recommended"
+    ],
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "ecmaVersion": 12,
+      "sourceType": "module"
+    },
+    "plugins": ["react", "prettier"],
```

Comment:
If adding prettier here, probably can add it to the vscode extension suggestion

Reply:
but we are formating with eslint also I have disabled prettier. so should I add it?